install/playbooks/roles/user-setup/tasks/main.yml

```diff
@@ -97,16 +37,12 @@
   loop_control:
     loop_var: dir
 
-- name: Create the custom configuration directory for each user
-  file:
-    path: '/home/users/{{ user.uid }}/.config/homebox'
-    owner: '{{ user.uid }}'
-    group: users
-    state: directory
-    recurse: true
+- name: Create the users home folders from /etc/skel
+  tags: accounts
+  command: 'mkhomedir_helper {{ user.uid }} 077 /etc/skel'
   with_items:
     - '{{ users }}'
-    - uid: postmaster
+    - '{{ ldap.postmaster }}'
```

Comment:
If you keep user.uid above, you need to keep the line uid: postmaster, or an error may occurs, I think.

Reply:
The `config/defaults.yml` file is also added as a var file for the playbook, so `ldap.postmaster` is accessible. The `ldap.postmaster` object contains the `uid`. (And I thought it was configurable, but it is not.)